.github/README.md

```diff
@@ -0,0 +1,61 @@
+﻿# GitHub Actions Workflow Configuration
+
+This template contains 2 workflows:
+1. `main.yml` - This workflow is triggered on pull requests to the `main`, `master` or `develop` branches OR on pushes to `main`, `master` or `develop` branches.
+2. `release.yml` - This workflow is triggered on pushes to `main` or `master` branches.
+
+## Configuration
+For the workflows to run, you need to configure some secrets in your repository settings as well as make some string replacements in the workflow files.
+
+### Environment Variables
+In `release.yml`, you need to inspect the environment variable definitions and insert your own values for Godot Asset Lib and Itch.io, respectively.
+
+### String Replacements
+In both `main.yml` and `release.yml`, you need to replace, just with the rest of this template repository, replace all occurrences of `ADDON_NAME` with the name of your addon! Please make sure there are no spaces, and only alphanumeric values, dashes and underscores in this sting!.
+
+### Secrets
+#### PAT
+Because we're committing back from the workflow to the repo, you need to provide a "Personal Access Token" or "PAT" to the workflow. This is a token that allows the workflow to commit back to the repo. You can create a PAT in your GitHub settings. [See the documentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).
+This PAT needs to have the `repo` scope.
+
+Once you have created this PAT, you need to add it to the repository secrets as `PAT`.
+
+#### Godot Asset Library
+If you want to publish to the Godot Asset Library, you need to add the following secrets to your repository:
+
+* `ASSET_STORE_PASSWORD` - The password for your Godot Asset Library account
```

Comment:
In the `release.yml` file, it is called `ASSET_LIB_PASSSWORD`

Reply:
Fixed. Thanks!